main.tf

```diff
@@ -79,6 +79,30 @@ locals {
       volumesFrom = []
     }
   ]
+  xray_container = [{
+    name       = "${var.app_name}-xray"
+    image      = "public.ecr.aws/xray/aws-xray-daemon:3.x"
+    essential  = true
+    privileged = false
+    portMappings = [{
+      containerPort = 2000
+      hostPort      = null
+      protocol      = "udp"
+    }]
+    logConfiguration = {
+      logDriver = "awslogs"
+      options = {
+        awslogs-group         = "${local.cloudwatch_log_group_name}-xray"
```

Comment:
Do you need to specify this log group (as in create your own custom log group) or is it already created? I could be wrong, but if it is automatically created, it'll use the default log retention time which will be reset by the DivvyCloud policies (after sending an error/warning email to account owner).

Reply:
I'm not sure what you mean. Are you suggesting not having a log group and having Divvy create it?

Reply:
No, I'm suggesting that we might have to manually create our own cloudwatch log group in this terraform module so that Divvy doesn't get mad and automatically change the log retention time.

Reply:
Oh. This pattern of creating the log group matches what we do for the user-provided definitions. And I'm pretty sure those already have a default length of four months (based on looking at the Payments services).

Reply:
OK, just making sure.